frontend/views/containers/contributions/IncomeForm.vue

```diff
@@ -0,0 +1,219 @@
+<template lang='pug'>
+modal-template(
+  ref='modal'
+  :isactive='true'
+  @close='cancelForm'
+)
+  form(
+    @submit.prevent='verifyForm'
+    novalidate='true'
+  )
+    i18n(tag='h2') Income Details
+
+    .field
+      i18n.label(tag='label') Do you make at least {{fakeStore.mincomeFormatted}} per month?
+
+      .radio-wrapper
+        input.radio(
+          name='v-model'
+          type='radio'
+          v-model='form.option'
+          @change='resetFormVerify'
+        )
+        i18n No, I don&apos;t
+
+      .radio-wrapper
+        input.radio(
+          name='v-model'
+          type='radio'
+          v-model='form.option'
+          @change='resetFormVerify'
+        )
+        i18n  Yes, I do
+
+      p.error(v-if='$v.form.option.$error')
+        i18n {{infoRequired}}
+
+    .field(v-if='needsIncome')
+      i18n.label(tag='label')  What's your monthly income?
+
+      .input-combo
+        input.input(
+          type='number'
+          placeholder='New amount'
+          :class='{"error": inputIncomeError }'
+          @input='verifyInputIncome'
+          :value='form.income'
+          :placeholder='L("amout")'
+        )
+        label USD
+
+      p.error(v-if='inputIncomeError') inputIncomeError
+
+      p.help(v-else)
+        text-who(:who='["Rick", "Carl", "Kim"]')
+        i18n will ensure you meet the mincome
```

Comment:
1. This sentence isn't being built properly, you can't have i18n being split up like this, it will lose context and becomes untranslatable.
2. When you put in the variable (using `{`), use the data from the vuex store instead of dummy data since it's now available